src/utils/field-value-converter.ts

```
@@ -0,0 +1,140 @@
+    
+    try {
+      switch (type) {
+        case 'text':
+          return {
+            value: value?.toString(),
+            convertedValue: value?.toString(),
+            columnName: 'textValue'
+          };
+
+        case 'number':
+          const numValue = parseFloat(value);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)